web/gui/dashboard_info.js

```diff
@@ -1387,8 +1387,119 @@ netdataDashboard.context = {
     // ------------------------------------------------------------------------
     // IPv4
 
+    'ipv4.packets': {
+        info: '<p>IPv4 packets statistics for this host.</p>'+
+        '<p><b>Received</b> - packets received by the IP layer. '+
+        'This counter will be increased even if the packet is dropped later. '+
+        '<b>Sent</b> - packets sent via IP layer, for both single cast and multicast packets. '+
+        'That counter does not include any packets counted in <b>Forwarded</b>. '+
```

Comment:
I think you wanted to use `This counter` instead `That counter`, because you are talking about `Sent`. :thinking: